include/rtc/rtc.h

```diff
@@ -252,15 +252,22 @@ RTC_C_EXPORT int rtcGetDataChannelReliability(int dc, rtcReliability *reliabilit
 
 // Track
 
+typedef struct {
+	int id;
+	char *uri;
+} rtcExtMapInit;
+
 typedef struct {
 	rtcDirection direction;
 	rtcCodec codec;
 	int payloadType;
 	uint32_t ssrc;
 	const char *mid;
-	const char *name;    // optional
-	const char *msid;    // optional
-	const char *trackId; // optional, track ID used in MSID
+	const char *name;             // optional
+	const char *msid;             // optional
+	const char *trackId;          // optional, track ID used in MSID
+	const rtcExtMapInit **extMaps; // optional
```

Comment:
If you add an `extMapsCount` field instead of relying on a NULL terminator, you could remove the double pointer and make this a simple array of `rtcExtMapInit`, which is simpler to initialize on user side.